From 314744b42802d3f24ba7a97be30c573fd3db25b1 Mon Sep 17 00:00:00 2001
From: "duncan.macleod" <duncan.macleod@ligo.org>
Date: Mon, 13 Nov 2023 15:19:43 +0000
Subject: [PATCH 4/5] Python: use double quotes in ctypesgen command

so that windows groups the options properly

(cherry picked from commit cc17a056e0e7dc1ec7f0b2e93f514e700a94744c)
---
 Python/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Python/CMakeLists.txt b/Python/CMakeLists.txt
index b88ff78..9789e87 100644
--- a/Python/CMakeLists.txt
+++ b/Python/CMakeLists.txt
@@ -83,7 +83,7 @@ if (ENABLE_PYTHON)
     add_custom_command(
         OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/framel.py
         COMMENT "Building the python ctypes binding"
-        COMMAND ${CTYPESGEN} --cpp '${CMAKE_C_COMPILER} -E'
+        COMMAND ${CTYPESGEN} --cpp "${CMAKE_C_COMPILER} -E"
             --include-symbols=stdout
             -l ${LIBFRAMEL_NAME}
             --compile-libdir ${LIBFRAMEL_LIBDIR}
-- 
2.39.2

